### DISCUSSION TITLE
[Request] Toggle Mobgriefing in Claims

### DISCUSSION THREAD

**Discussion by META-mahn** [GriefPrevention/GriefPrevention#72] [Ideas]
Hello Github people,

Recently on a server (named Dawnhaven) I've discovered the need for mobgriefing to be on.
Why? Because a recent machine I've been creating in my SP test worlds used Wither mobgriefing in order to break cobblestone, fed into a Wither cage by pistons.
While I (and friends) are trying to convince the server owner to turn mobgriefing off (because the spawn rate of hostile mobs on the server is really low), I was wondering if you all could code it so there could be a /claimmobgrief command that turns mobgriefing on and off in that claim.

Thanks, 
Meta

**Comment by RoboMWM** [comment]
Potentially https://www.spigotmc.org/threads/griefprevention-flags-from-the-creator-of-griefprevention-paid.82302/page-24#post-2079432 might provide such an option? Also try /claimexplosions

**Comment by META-mahn** [comment]
I've tried toggling off /claimexplosions and running a Wither on Normal into cobblestone. Doesn't work.
Will contact the server owners about GriefProt flags though.

**Comment by META-mahn** [comment]
UPDATE: Just checked GriefProt flags myself, there doesn't appear to be a mobgrief flag available.

**Comment by RoboMWM** [comment]
`/claimexplosions` will function like this in version 20 (will be known as toggling "natural grief.")
